website/docs/concepts/how-caching-works.md

```diff
@@ -199,7 +199,6 @@ option for the task runner in `nx.json`:
   "tasksRunnerOptions": {
     "default": {
       "options": {
-        "cacheableOperations": ["build", "test"],
```

Comment:
This whole config is outdated too, cache directory is set at the top level, no needs for the object

Reply:
I've removed this whole section. We're pretty clear that it's Nx doing the caching in the docs and users can find the Nx docs if they need to do advanced things like changing the cache location.

I think we should minimize the amount Nx-specific docs on the Lerna site as they will get out of date quickly.